ipatests/test_webui/test_pwpolicy.py

```diff
@@ -54,4 +51,214 @@ def test_crud(self):
         Basic CRUD: pwpolicy
         """
         self.init_app()
-        self.basic_crud(ENTITY, DATA)
+        self.basic_crud(pwpolicy.ENTITY, pwpolicy.DATA)
+
+    @screenshot
+    def test_misc(self):
+        """
+        various test cases covered in one place
+        """
+        # Basic requirement : create user group for test
+        self.init_app()
+        self.navigate_to_entity(group.ENTITY)
+        self.add_record(group.ENTITY, [group.DATA, group.DATA2, group.DATA3,
+                                       group.DATA4, group.DATA5, group.DATA6,
+                                       group.DATA_SPECIAL_CHAR_GROUP])
+
+        # add then cancel
+        self.add_record(pwpolicy.ENTITY, pwpolicy.DATA1, dialog_btn='cancel')
+
+        # test add and add another record
+        self.add_record(pwpolicy.ENTITY, [pwpolicy.DATA1, pwpolicy.DATA2])
+
+        # test add and edit record
+        self.add_record(pwpolicy.ENTITY, pwpolicy.DATA,
+                        dialog_btn='add_and_edit')
+
+        # test add password policy for special characters group
+        self.add_record(pwpolicy.ENTITY, pwpolicy.DATA_SPECIAL_CHAR,
+                        delete=True)
+
+        # empty group and priority (requires the field)
+        self.navigate_to_entity(pwpolicy.ENTITY)
```

Comment:
is the navigation needed? Shouldn't we be already on the page?

Reply:
Thanks for catching this. I have removed.